paraview/generate_dist_javascript.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+npm install paraviewweb  --save
```

Comment:
I assume some of the lines will generate the file we removed, could you add some comments to explain what the purpose of each command?